### PR TITLE
Default active vulnerability management to false

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Flag out active vulnerability management by default, but allow customers to modify it with a flag.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Scale test

### CODE REVIEW

**Comment by connorgorman** [comment]
@c-du @md2119 @theencee No need to review the UI changes as those were previously reviewed

**Review by md2119 — APPROVED** [review]

File: central/graphql/resolvers/active_state.go [hunk 1]
```diff
@@ -18,6 +19,8 @@ const (
 	Inactive
 	// Active means the vulnerability or component is active.
 	Active
+	// Disabled means the feature is disabled
+	Disabled
```
[the remark this answers is not in the document]
Comment (connorgorman) [reply central/graphql/resolvers/active_state.go]: Done

**Pull request merged**